src/System.IO.FileSystem/tests/Directory/Delete.cs

```diff
@@ -250,6 +251,33 @@ public void RecursiveDeleteWithTrailingSlash()
         }
 
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [OuterLoop("This test is very slow.")]
```

Comment:
Have you tried running the outerloop jenkins machines to see just how slow it is?